lib-vau-csharp/VauClient.cs

```diff
@@ -12,87 +12,191 @@
  * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
  * See the License for the specific language governing permissions and
  * limitations under the License.
- *
- * For additional notes and disclaimer from gematik and in case of changes by gematik find details in the "Readme" file.
  */
 
 using lib_vau_csharp.data;
+using lib_vau_csharp.util;
 using lib_vau_csharp.exceptions;
+
 using System;
 using System.Collections.Generic;
+using System.Diagnostics;
 using System.Linq;
 using System.Net.Http;
 using System.Text;
+using System.Text.Json;
+using System.Text.Json.Serialization;
 using System.Threading.Tasks;
 
-
 namespace lib_vau_csharp
 {
+    [DebuggerDisplay("ConnectionId: {ConnectionId}")]
     public class VauClient
```

Comment:
I feel like this class should really be named `VauSession`. Thoughts?

Reply:
I don't think anything else needs to be changed. My thinking is, that from a consumer perspective, "session" makes it clear that the object has a finite lifetime, since a connection to the VAU is only valid for 24 hours, whereas "client" does not communicate this (e.g. a HttpClient can be used indefinitely). 
I don't intend to propose this change within this PR, just wanted to get some feedback, so I'll leave it as is here.